starter-code/server.js

```diff
@@ -45,19 +53,30 @@ app.post('/articles', function(request, response) {
 
   function queryTwo() {
     client.query(
-      ``, // TODO: Write a SQL query to retrieve the author_id from the authors table for the new article
-      [], // TODO: Add the author name as data for the SQL query
+      `SELECT author_id FROM authors WHERE author = '${request.body.author}' AND "authorUrl" = '${request.body.authorUrl}';`,
+       // DONE: Write a SQL query to retrieve the author_id from the authors table for the new article
+       // est: 10min act: 40min
+       // DONE: Add the author name as data for the SQL query
+       // est: 1min act: 1min
       function(err, result) {
         if (err) console.error(err)
-        queryThree(result.rows[0].author_id) // This is our third query, to be executed when the second is complete. We are also passing the author_id into our third query
+        queryThree(result.rows[0].author_id) // This is our third query, to be executed when the second is complete.
+        // We are also passing the author_id into our third query
       }
     )
   }
 
   function queryThree(author_id) {
     client.query(
-      ``, // TODO: Write a SQL query to insert the new article using the author_id from our previous query
-      [], // TODO: Add the data from our new article, including the author_id, as data for the SQL query.
+      `INSERT INTO articles ($1, title, category, "publishedOn", body), VALUES ($2, $3, $4, $5, $6);`,
```

Comment:
Looks like you're one number off here...you can directly say "author_id" in the first parens (column names), and then fix your references in the values parens